beacon-chain/chaintest/main.go

```diff
@@ -77,19 +88,28 @@ func runTests(tests []interface{}, sb *backend.SimulatedBackend) error {
 			log.Infof("Test Suite: %v", typedTest.TestSuite)
 			for _, testCase := range typedTest.TestCases {
 				if err := sb.RunForkChoiceTest(testCase); err != nil {
-					return fmt.Errorf("chain test failed: %v", err)
+					return fmt.Errorf("fork choice test failed: %v", err)
 				}
 			}
 			log.Info("Test PASSED")
 		case *backend.ShuffleTest:
 			log.Infof("Title: %v", typedTest.Title)
 			log.Infof("Summary: %v", typedTest.Summary)
-			log.Infof("Test Suite: %v", typedTest.TestSuite)
-			log.Infof("Fork: %v", typedTest.Fork)
-			log.Infof("Version: %v", typedTest.Version)
+			log.Infof("Fork: %v", typedTest.Forks)
+			log.Infof("Config: %v", typedTest.Config)
+			config, ok := configs[string(typedTest.Config)]
+			if !ok {
+				return fmt.Errorf("no config file found for test")
```

Comment:
no need for formatting if there is no variable to log

Reply:
fixed